go.mod

```diff
@@ -15,46 +15,51 @@
 
 module github.com/apache/spark-connect-go/v1
 
-go 1.19
+go 1.21
```

Comment:
This assumes #20 will be accepted and merged first, otherwise we can change this back to 1.19